pycript/execution.py

```diff
@@ -16,6 +16,7 @@ def execute_command(selectedlang, path, data, headervalue=None):
         with tempfile.NamedTemporaryFile(delete=False, mode='w') as temp_file:
             json.dump(content, temp_file)
             temp_file_path = temp_file.name
+            print(temp_file_path)
```

Comment:
unwanted, temp files are always deleted during execution process, printing file name make no sense and not useful as after the print and execution file is already deleted